### ISSUE TITLE
GlusterFS StorageClass parameter is invalid when deploying both glusterfs and glusterfs_registry

### ISSUE THREAD

**Issue by screeley44** [openshift/openshift-ansible#4437]
@jarrpa 
```
parameters:
    resturl: http://localhost:8080:8081
```

```
Events:
  FirstSeen	LastSeen	Count	From				SubObjectPath	Type		Reason			Message
  ---------	--------	-----	----				-------------	--------	------			-------
  1m		10s		5	persistent-volume-controller			Warning		ProvisioningFailed	Failed to provision volume with StorageClass "glusterfs-storage": glusterfs: create volume err: error creating volume Post http://localhost:8080:8081/volumes: dial tcp: too many colons in address localhost:8080:8081.
```

This also happens on just deploying glusterfs ...see storageclass output from #4440 

Using @jarrpa branch glusterfs-fixes

**Comment by jarrpa** [comment]
Oops, durr. Easy fix. :)

**Comment by jarrpa** [comment]
Should be fixed by #4484, please verify.

**Comment by screeley44** [comment]
fixed
